src/_data/header2.yml

```diff
@@ -0,0 +1,16 @@
+- name: Enterprise
+  href: enterprise
+  primary: true
+  landingurl: true
+- name: Docs
+  href:
+  docsurl: true
+- name: GitHub
+  href: https://github.com/tilt-dev/tilt
+  githubStars: 3.5k
```

Comment:
hahah nice work

Reply:
Haha, I figure I could make a new story to make this properly update...